### ISSUE TITLE
notify command's run steps run in the default executor shell

### ISSUE THREAD

**Issue by m0sa** [CircleCI-Public/slack-orb#344]
## Orb version:

4.9.3

## What happened:

The notify command tries to execute it's bash script in the default shell. I have a executor set up as:

```
version: 2.1
orbs:
  slack: circleci/slack@4.9.3
executors:
  dotnet-windows:
    machine:
      image: windows-server-2019-vs2019:stable
    resource_class: windows.large
    shell: powershell.exe -ExecutionPolicy Bypass
jobs:
  example:
    executor: dotnet-windows
  steps:
    - slack/notify:
        ...
        # fails because the steps in the notify command do not explicitly specify `shell: bash`
```

## Expected behavior:

Command explicitly specifies it's steps should run in a bash shell.

**Comment by EricRibeiro** [comment]
Hi @m0sa 👋 

Looks like you are trying to run a job with the Windows Executor. This orb does not officially support Windows. From the [FAQ](https://github.com/CircleCI-Public/slack-orb/wiki/FAQ):

```
This orb is written entirely in Bash, designed for Linux to be run in the shell. While there is a bash.exe available in the CircleCI Windows VMs, not all of the installed tools are exactly the same as their true GNU linux counterparts. Specifically, the version of sed found within Windows (technically the same version) is not behaving exactly the same way in Windows as it does within Linux/Mac.

Alternative integrations will be required for Windows based builds. Consider connecting our [CircleCI Webhooks](https://circleci.com/docs/2.0/webhooks/) with an automation service which connects to Slack.
```

**Comment by m0sa** [comment]
Thanks, didn't see this.

**Comment by EricRibeiro** [comment]
Hey, @m0sa 👋 

Support for Windows was implemented in [v4.12.0](https://github.com/CircleCI-Public/slack-orb/releases/tag/v4.12.0). If you want to give it a shot, make sure to choose `bash.exe` as your [shell in the Windows executor](https://circleci.com/docs/using-windows/#specifying-a-shell-with-the-windows-executor).

Let me know how it goes!
